### DISCUSSION TITLE
[공지사항] 공지사항 확인하면 댓글 달기!!

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#66] [00_공지사항]
여러분 안녕하세요!

멘토 유혁준입니다.

여러분들 과제하느라 참 고생 많으십니다. 나날히 발전하는 모습 참 보기 좋구요!

Discussion 에서 제가 올리는 공지사항들을 여러분들이 확인하는 것을 멘토들도 모두 체크하려합니다.
원활한 수업 진행을 위해서 하는 것이기 때문에 양해 부탁드려요.

공지사항 글이 올라오면 항상 댓글로 확인! 이라고 댓글을 달아주시길 바랄게요.
질문 있으면 질문 하셔도 됩니다.

1. github 가입한 이메일 들어가서 메일오면 알림오도록 설정하기
2. Notion App 다운로드 후, 업데이트 알림 켜두기

공지사항 및 업데이트는 항상 확인해주시면서 과제와 수업을 들으시길 바랄게요

감사합니다.

**Comment by lsj0822** [comment]
확인했습니다!

**Comment by jiwoo4424** [comment]
확인!

**Comment by ghost** [comment]
확인~!!

**Comment by NA-DK** [comment]
확인 했습니다.

**Comment by dlrudgnsdlek** [comment]
확인 했습니다
